eazye.go

```diff
@@ -106,6 +106,50 @@ func GenerateSince(info MailboxInfo, since time.Time, markAsRead, delete bool) (
 	return generateMail(info, "", &since, markAsRead, delete)
 }
 
+// MarkAsUnread will set the UNSEEN flag on a supplied slice of UIDs
+func MarkAsUnread(info MailboxInfo, uids []uint32) error {
+
+	client, err := newIMAPClient(info)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		client.Close(true)
+		client.Logout(30 * time.Second)
+	}()
+	for _, u := range uids {
+		fmt.Println("Setting unseen on", u)
```

Comment:
Mind dropping the debug statement here and below?

Reply:
Oops right you are, done!